collect_app/src/main/res/layout/server_auth_dialog.xml

```diff
@@ -37,9 +37,8 @@
         android:layout_marginLeft="20dip"
         android:layout_marginRight="20dip"
         android:scrollHorizontally="true"
-        android:inputType="textNoSuggestions"
-        android:autoText="false"
-        android:capitalize="none"
+        android:inputType="textFilter|textNoSuggestions"
+        android:textAllCaps="false"
```

Comment:
Do we really need that? Why? I guess you have added this instead of `android:capitalize="none"` ?
I think it's false by default and if we want to change something we can use:

> android:inputType="textCapCharacters|textCapSentences|textCapWords"